README.md

```diff
@@ -0,0 +1,52 @@
+# Prometheus OVN Exporter Snap
+
+This repository contains the source for [prometheus-ovn-exporter](https://snapcraft.io/prometheus-openstack-exporter), an unofficial snap of the [OVN Exporter](https://github.com/greenpau/ovn_exporter) for
```

Comment:
Just curious about what to read into the word `unofficial` here, unofficial as in not endorsed by upstream? From distribution point of view it is owned by the `Canonical` publisher, which kind of is "official".

Would there be a different word that is more fitting?

Reply:
To be honest, I copied this wording from another snap repo. I guess it stems from the original push that snap packaging should be in upstream git repos. I'm happy to remove it.

Reply:
Pushed a new revision with updated wording